Add a note that if list pipelines exceeds 10k, Kibana faces performan… (#131992)
elastic/kibana: docs/api/logstash-configuration-management/list-pipeline.asciidoc

experimental[] List all centrally-managed Logstash pipelines.

> [IMPORTANT]
> ====
> Limit your number of pipelines to 10k or fewer. When the number of pipelines surpasses 10k, you may see performance issues on Kibana.
> ====

Reviewer: ```suggestion
IMPORTANT: Limit the number of pipelines to 10k or fewer. As the number of pipelines nears and surpasses 10k, you may see performance issues on Kibana.
```

Reviewer: You can use the basic (not block) format for a single paragraph admonition. 

What do you think about saying "As the number of pipelines nears and surpasses..." to indicate that 10k is not an absolute?

Author: Oh thanks! TIL: didn't know without block it works. And thanks for revising the message.
